api/python/provisioner/cli_parser.py

```diff
@@ -141,8 +141,13 @@ def parse_args(args=None, commands: Optional[List] = None):
 
         # TODO description and help strings
         for cmd_name, cmd in commands.items():
+
+            desc = getattr(cmd, '_description', None)
```

Comment:
you may optimize that and use default value instead of `None`

Reply:
comment is addressed